src/transformers/models/layoutxlm/tokenization_layoutxlm_fast.py

```diff
@@ -34,7 +34,7 @@
 )
 from ...tokenization_utils_fast import PreTrainedTokenizerFast
 from ...utils import logging
-from ..xlm_roberta.tokenization_xlm_roberta import (
+from ..xlm_roberta.tokenization_xlm_roberta_fast import (
```

Comment:
Here, I think it's a bug fix because the fast version of `LayoutXLMTokenizerFast` needs the info from the fast files.